src/Conf.ml

```diff
@@ -1549,7 +1547,7 @@ let root =
   Option.map !root ~f:Fpath.(fun x -> v x |> to_absolute |> normalize)
 
 let disable_outside_detected_project =
```

Comment:
It will increase the size of the diff slightly, but I think this should also be renamed.